autoload/nim/suggest.vim

```diff
@@ -41,8 +41,10 @@ function! s:NewInstance(project, file)
     endif
   endfunction
 
+  let nscmd = exists('g:nim_nimsuggest_path') ? g:nim_nimsuggest_path : "nimsuggest"
```

Comment:
```suggestion
  let nscmd = exists('g:nim_nimsuggest_cmd') ? g:nim_nimsuggest_cmd : "nimsuggest"
```

I personally think this should be `_cmd`, as this is what we use to call nimsuggest, and it could be any executable in `PATH` and not just a full path to nimsuggest.

Also, please modify your commit description to fit the changes.

Reply:
(that a file path, not a command, but ok)